[BUG] Type mismatch: inferred type is Boolean? but Boolean was expected null-safety  issue

When I'm trying to build project with photo manager, I receive error as follows. 

**To Reproduce**
Steps to reproduce the behavior:
Build project with PhotoManager 2.0.2 Included

```dart
import 'package:flutter/material.dart';
import 'package:gallery_app/widgets/app_bar.dart';
import 'package:gallery_app/widgets/image_grid.dart';
import 'package:photo_manager/photo_manager.dart';

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key}) : super(key: key);

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  bool permissionsOk = false;
  void requestPermissionAndCgeckIfGranted() async {
     final PermissionState _ps = await PhotoManager.requestPermissionExtend();
     if (_ps == PermissionState.authorized) {
       permissionsOk = true;
     }
  }

  @override
  void initState() {
    requestPermissionAndCgeckIfGranted();
    super.initState();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: buildAppBar(),
      body: ImageGrid(),
    );
  }
}
```

**Flutter version**
<!-- 
Flutter 2.13.0-0.0.pre.74 
 -->

**Smartphone (please complete the following information):**
 - Device: [Xiaomi]
 - OS: [Android]


**Log**

```
opt/flutter/.pub-cache/hosted/pub.dartlang.org/photo_manager-2.0.0/lib/src/internal/image_provider.dart:111:25: Warning: Operand of null-aware operation '?.' has type 'PaintingBinding' which excludes null.
 - 'PaintingBinding' is from 'package:flutter/src/painting/binding.dart' ('/opt/flutter/packages/flutter/lib/src/painting/binding.dart').
        PaintingBinding.instance?.imageCache?.evict(key);
                        ^
e: /opt/flutter/.pub-cache/hosted/pub.dartlang.org/photo_manager-2.0.0/android/src/main/kotlin/com/fluttercandies/photo_manager/core/PhotoManagerPlugin.kt: (107, 34): Type mismatch: inferred type is Boolean? but Boolean was expected
e: /opt/flutter/.pub-cache/hosted/pub.dartlang.org/photo_manager-2.0.0/android/src/main/kotlin/com/fluttercandies/photo_manager/core/PhotoManagerPlugin.kt: (107, 39): Type mismatch: inferred type is Boolean? but Boolean was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':photo_manager:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

--- a/android/src/main/kotlin/com/fluttercandies/photo_manager/core/PhotoManagerPlugin.kt
+++ b/android/src/main/kotlin/com/fluttercandies/photo_manager/core/PhotoManagerPlugin.kt
@@ -102,7 +102,7 @@ class PhotoManagerPlugin(
                 true
             }
             Methods.log -> {
-                LogUtils.isLog = call.arguments()
+                LogUtils.isLog = call.arguments() ?: false
                 resultHandler.reply(1)
                 true
             }

